config-connector/solutions/projects/helm/project-services/README.md

```diff
@@ -0,0 +1,97 @@
+# Project Services
+
+==================================================
+
+## NAME
+
+  project-services
+
+## SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a desired project.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the project-services folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/projects/helm/project-services
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with [Config Connector installed using a GKE Workload Identity](https://cloud.google.com/config-connector/docs/how-to/install-upgrade-uninstall#workload-identity).
+
+1. Cloud Resource Manager API enabled in the project where Config Connector is installed.
+
+1. The "cnrm-system" service account that has the `roles/serviceusage.serviceUsageAdmin` or `roles/owner` role in your desired project (it doesn't need to be the project managed by Config Connector).
```

Comment:
Could you remove `(it doesn't need to be the project managed by Config Connector)`? The desired project here must be the project managed by Config Connector, but it could be different from the project where Config Connector is installed.